pom.xml

```diff
@@ -276,6 +276,13 @@
     <spark.test.docker.keepContainer>false</spark.test.docker.keepContainer>
     <spark.test.docker.removePulledImage>true</spark.test.docker.removePulledImage>
 
+    <!-- Version used in Connect -->
+    <connect.guava.version>31.0.1-jre</connect.guava.version>
```

Comment:
Why call `connect.guava.version`? Because the name `guava.version` already exists in this file.